pkg/network/ovn_kubernetes.go

```diff
@@ -30,6 +30,7 @@ const OVN_NB_RAFT_PORT = "9643"
 const OVN_SB_RAFT_PORT = "9644"
 const CLUSTER_CONFIG_NAME = "cluster-config-v1"
 const CLUSTER_CONFIG_NAMESPACE = "kube-system"
+const OVN_CERT_CN = "ovnkube-master"
```

Comment:
The certificate is used for more than just the master - every component uses it! That's ovn-northd, nbdb, sbdb, ovnkube-node, and ovn-controller.

I'd just call it "ovn" :p